src/interfaces/src/plugin/types.h

```diff
@@ -34,6 +34,11 @@ typedef struct interfaces_interface_state interfaces_interface_state_t;
 typedef struct interfaces_interface_state_hash_element interfaces_interface_state_hash_element_t;
 typedef struct interfaces_interface_hash_element interfaces_interface_hash_element_t;
 
+enum interfaces_interfaces_interface_enable {
```

Comment:
If the enum is being used, set enabled variant to 1 and disabled to 0 to match the boolean YANG data format:
```c
enum interfaces_interfaces_interface_enable {
    interfaces_interfaces_interface_enable_disabled = 0,
    interfaces_interfaces_interface_enable_enabled = 1,
};
```